addon/helpers/moment-format.js

```diff
@@ -13,15 +14,25 @@ export default BaseHelper.extend({
     this.recompute();
   }),
 
-  compute: computeFn(function(params, { locale, timeZone }) {
+  compute: computeFn(function(params, { locale, timeZone, interval }) {
+    this.clearTimer();
+
     const length = params.length;
 
-    if (length > 3) {
-      throw new TypeError('ember-moment: Invalid Number of arguments, expected at most 3');
+    if (length > 4) {
+      throw new TypeError('ember-moment: Invalid Number of arguments, expected at most 4');
+    }
+
+    if (interval) {
+      this.timer = setTimeout(runBind(this, this.recompute), parseInt(interval, 10));
     }
 
     let format;
     const args = [];
+    
+    if(params[0] === 'now') {
```

Comment:
This change is out of scope but I'd be happy to consider it in the future just not in this changeset